bika/lims/browser/analysisrequest/analysisrequests.py

```diff
@@ -709,7 +702,8 @@ def folderitem(self, obj, item, index):
         # append partition UIDs of this AR as children
         item["children"] = obj.getDescendantsUIDs or []
 
-        return item
+        # Call the folderitem method from the base class
+        return BikaListingView.folderitem(self, obj, item, index)
```

Comment:
Calling the super class implementation *after* our own implementation is not a good approach. Imagine if we decide in half a year to set one of the injected keys in the base implementation...
A better approach is to call the subscribers here:
https://github.com/senaite/senaite.core.listing/blob/master/src/senaite/core/listing/view.py#L872
and here:
https://github.com/senaite/senaite.core.listing/blob/master/src/senaite/core/listing/view.py#L1071

Reply:
Ummm, you're right. Let me update the thing